### PR TITLE
fix: add process.exit for generators

### PR DESCRIPTION
**Pull request by aaqilniz**
Running `lb4 discover` with `--config` option causes it to get stuck. The lb4 discover works fine with other options like  `--dataSource` & `--schema` but it gets stuck when we run it with `--config`. The discoverer successfully generates the models but doesn't terminate. 

This PR fixes that

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/cli/test/fixtures/copyright/single-package/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-// Copyright IBM Corp. and LoopBack contributors 2020. All Rights Reserved.
-// Node module: @loopback/cli
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
+// Copyright ACME Inc. 2020,2023. All Rights Reserved.
```
Comment (dhmlau) [inline packages/cli/test/fixtures/copyright/single-package/index.js:1]: @aaqilniz, could you please revert this change?

**Review by dhmlau — CHANGES_REQUESTED** [review]
@aaqilniz, thanks for the fix. I wonder if there's particular reason we need to update the copyright statement? 
For the new files, we don't necessarily need to have copyright statements in each file. But for any existing one, we cannot remove any owners. Thanks.

File: packages/cli/test/fixtures/copyright/single-package/index.js (hunk 1 above)
Comment (aaqilniz) [reply packages/cli/test/fixtures/copyright/single-package/index.js]: Hi, @dhmlau. The statements were mistakenly modified. I have reverted the changes.

**Comment by aaqilniz** [comment]
Hi, @dhmlau. I've reverted the changes related to the copyright statements. Can you please have a look at the PR once again?

**Comment by dhmlau** [comment]
Thanks @aaqilniz !
